tests/spec/core/markdown-spec.js

```diff
@@ -372,7 +372,7 @@ describe("Core - Markdown", () => {
   });
 
   it("properly indents <pre> contents with no block indentation", async () => {
-    const idl = `function getAnswer() {\n  return 42;\n}`;
+    const idl = `function getAnswer() {\n  return 42;\n}\n`;
```

Comment:
@saschanaz I'm not sure of the implications of this. Can you validate this "fix"?

Reply:
It seems to match the GitHub Markdown behavior. Since the dangling whitespaces are ignored, I see no problem.

Reply:
Thanks!